vuepress/docs/next/tutorials/ecr/github-actions-workflow.md

```diff
@@ -0,0 +1,107 @@
+---
+sidebarDepth: 2
+---
+
+# Add a GitHub Actions CI Workflow
+
+## Overview
+This tutorial shows how to use the JHipster `ci-cd` sub-generator to quickly create a basic GitHub Actions Workflow for the microservices and micro frontends generated by the Entando Component Generator.
+
+## Prerequisites
+* An existing project with a GitHub repository. See the [Entando Component Generator](../backend-developers/generate-microservices-and-micro-frontends.md#overview) to create a new one.
+* [GitHub Actions](https://github.com/features/actions). These are enabled by default without additional configuration. Please note public repositories currently qualify for unlimited usage but private repositories can encounter usage restrictions.
+* Use the [Entando CLI](../../docs/reference/entando-cli.md#check-environment) to verify you have the command line prerequisites in place for this tutorial (e.g. npm, git, JHipster).
+
+## Create a backend workflow
+
+1. Go to your main project folder in the shell
+2. Run the JHipster `ci-cd` subgenerator
+```bash
+ent jhipster ci-cd
+```   
+2. Select `GitHub Actions`
+3. Click ENTER to choose no `tasks/integrations` and generate the default workflow
```

Comment:
Workflow capitalized? (Comment also applies to all instances below...) Anthony and I had agreed on Workflow due to the fact that references were to a specific workflow, and Workflow is capitalized in the GitHub documentation... it's a gray area.

Reply:
I think uppercase when referring to the feature but lowercase when referring to a specific workflow. That seems to be how GitHub does it.
<img width="473" alt="image" src="https://user-images.githubusercontent.com/645546/147621451-6db3fd2c-c75f-4dfe-ac7a-f5c0d5417918.png">